keps/prod-readiness/sig-apps/3998.yaml

```diff
@@ -1,3 +1,5 @@
 kep-number: 3998
 alpha:
   approver: "@wojtek-t"
+beta:
+  approver: "@wojtek-t"
```

Comment:
@wojtek-t Could you take this KEP as a PRR approver?
Thanks in advance!

Reply:
Yes - will do